flatdata-generator/tests/generators/rust_expectations/enums/values.rs

```diff
@@ -7,6 +7,506 @@ pub enum EnumI8 {
     FooI8NegHex = -127,
     FooI8PosHex = 126,
     FooI8OneHex = 1,
+    #[doc(hidden)]
+    UnknownValueMinus126 = -126,
```

Comment:
I am not sure what is the benefit of generating all these value. In the end the user can't match them all, and they even do not know what to match against since the names are hidden in the docs.

Reply:
Without these some compilers will have undefined behaviour when transmuting these values unchecked.

This is essentially similar to the current practise of non-exhaustive Rust enums: https://github.com/rust-lang/rfcs/blob/master/text/2008-non-exhaustive.md